shared/teams/add-members-wizard/add-email.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
+import * as Styles from '../../styles'
+import {ModalTitle} from '../common'
+
+type Props = {
+  teamname: string
```

Comment:
Screens should generally take `teamID`s instead of names. Then you can map to a teamname using `Constants.getTeamMeta` inside your component.